.eslintrc.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
     "env": {
         "browser": true,
+        "node": true,
```

Comment:
I assume you need this for your plugin to pass linting. Ideally, we'd have a separate eslintrc for build tools, so we can't accidentally use nodejs apis in the app, which won't be supported.
Perhaps you can put an eslintrc in scripts and put your postcss directory there?

Reply:
done but I suppose we'd want nodejs support in the root directory as well (`vite*.config.js`). Maybe we ought to do the reverse i.e create an eslintrc without support for node in `/src`.